lldb/test/API/lang/swift/big_mpe/TestSwiftBigMPE.py

```diff
@@ -0,0 +1,35 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+
+class TestSwiftBigMPE(TestBase):
```

Comment:
would you mind spelling out MPE here and in the directory name? Might make it easier to grep for this later.